backend/api/chat_orchestrator.py

```
@@ -502,14 +468,21 @@ async def generate_scene_introduction_enhanced(self) -> str:
                         scene_title = summary_data.get('scene_data', {}).get('title', 'Previous Scene')
                         intro += f"• {scene_title}: Key insights and progress\n"
                     except (json.JSONDecodeError, TypeError) as parse_err:
-                        print(f"Unable to parse previous summary JSON: {parse_err}")
+                        debug_log(f"Unable to parse previous summary JSON: {parse_err}")
             
             return intro
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)